L1Trigger/Phase2L1GT/plugins/L1GTDeltaCut.h

```diff
@@ -97,30 +112,35 @@ namespace l1t {
         res &= maxDz_ ? dZ < maxDz_ : true;
       }
 
-      if (minDRSquared_ || maxDRSquared_) {
-        uint32_t dRSquared = dEta.value() * dEta.value() + dPhi.value() * dPhi.value();
+      uint32_t dRSquared = 0;
+      if (minDRSquared_ || maxDRSquared_ || minInvMassSqrOver2DRSqr_ || maxInvMassSqrOver2DRSqr_) {
+        dRSquared = dEta.value() * dEta.value() + dPhi.value() * dPhi.value();
         res &= minDRSquared_ ? dRSquared > minDRSquared_ : true;
         res &= maxDRSquared_ ? dRSquared < maxDRSquared_ : true;
       }
 
       res &= os_ ? obj1.hwCharge() != obj2.hwCharge() : true;
       res &= ss_ ? obj1.hwCharge() == obj2.hwCharge() : true;
 
-      int32_t lutCoshDEta = 0;
```

Comment:
Our script doesn't catch this yet (apparently), but you may get uninitialized variable complaints from build bot here in the long term. If it isn't breaking, it may be good to try and initialize this to something default to start. Same with the phi version.